+ source ../common/common_lib.sh
++ source /root/mugen-riscv/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start environmental preparation.'
+ message='Start environmental preparation.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start environmental preparation.'
Fri Feb  3 00:39:50 2023 - INFO  - Start environmental preparation.
+ DNF_INSTALL openvswitch
+ pkgs=openvswitch
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py install --pkgs openvswitch --node 1 --tempfile ''
+ tmpfile2=/tmp/tmp_xtj8qhz
+ '[' -z '' ']'
+ tmpfile=/tmp/tmp_xtj8qhz
+ service=ovsdb-server.service
++ date '+%Y-%m-%d %T'
+ log_time='2023-02-03 00:41:14'
+ LOG_INFO 'End of environmental preparation!'
+ message='End of environmental preparation!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End of environmental preparation!'
Fri Feb  3 00:41:15 2023 - INFO  - End of environmental preparation!
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start testing...'
+ message='Start testing...'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start testing...'
Fri Feb  3 00:41:16 2023 - INFO  - Start testing...
+ service ovsdb-server.service restart
oe_test_service_ovsdb-server.sh: line 32: service: command not found
+ CHECK_RESULT 127 0 0 'ovsdb-server.service restart failed'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='ovsdb-server.service restart failed'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'ovsdb-server.service restart failed'
+ LOG_ERROR 'ovsdb-server.service restart failed'
+ message='ovsdb-server.service restart failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'ovsdb-server.service restart failed'
Fri Feb  3 00:41:16 2023 - ERROR - ovsdb-server.service restart failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_service_ovsdb-server.sh line 33'
+ message='oe_test_service_ovsdb-server.sh line 33'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_service_ovsdb-server.sh line 33'
Fri Feb  3 00:41:17 2023 - ERROR - oe_test_service_ovsdb-server.sh line 33
+ return 0
+ service ovsdb-server.service stop
oe_test_service_ovsdb-server.sh: line 34: service: command not found
+ CHECK_RESULT 127 0 0 'ovsdb-server.service stop failed'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='ovsdb-server.service stop failed'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'ovsdb-server.service stop failed'
+ LOG_ERROR 'ovsdb-server.service stop failed'
+ message='ovsdb-server.service stop failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'ovsdb-server.service stop failed'
Fri Feb  3 00:41:17 2023 - ERROR - ovsdb-server.service stop failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_service_ovsdb-server.sh line 35'
+ message='oe_test_service_ovsdb-server.sh line 35'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_service_ovsdb-server.sh line 35'
Fri Feb  3 00:41:18 2023 - ERROR - oe_test_service_ovsdb-server.sh line 35
+ return 0
+ service ovsdb-server.service start
oe_test_service_ovsdb-server.sh: line 36: service: command not found
+ CHECK_RESULT 127 0 0 'ovsdb-server.service start failed'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='ovsdb-server.service start failed'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'ovsdb-server.service start failed'
+ LOG_ERROR 'ovsdb-server.service start failed'
+ message='ovsdb-server.service start failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'ovsdb-server.service start failed'
Fri Feb  3 00:41:19 2023 - ERROR - ovsdb-server.service start failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_service_ovsdb-server.sh line 37'
+ message='oe_test_service_ovsdb-server.sh line 37'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_service_ovsdb-server.sh line 37'
Fri Feb  3 00:41:19 2023 - ERROR - oe_test_service_ovsdb-server.sh line 37
+ return 0
+ service ovsdb-server.service status
+ grep 'Active: active (running)'
oe_test_service_ovsdb-server.sh: line 38: service: command not found
+ CHECK_RESULT 1 0 0 'ovsdb-server.service start failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='ovsdb-server.service start failed'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'ovsdb-server.service start failed'
+ LOG_ERROR 'ovsdb-server.service start failed'
+ message='ovsdb-server.service start failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'ovsdb-server.service start failed'
Fri Feb  3 00:41:20 2023 - ERROR - ovsdb-server.service start failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_service_ovsdb-server.sh line 39'
+ message='oe_test_service_ovsdb-server.sh line 39'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_service_ovsdb-server.sh line 39'
Fri Feb  3 00:41:20 2023 - ERROR - oe_test_service_ovsdb-server.sh line 39
+ return 0
+ grep -i 'fail\|error'
+ journalctl --since '2023-02-03 00:41:14' -u ovsdb-server.service
+ grep -v -i 'DEBUG\|INFO\|WARNING'
+ CHECK_RESULT 1 1 0 'There is an error message for the log of ovsdb-server.service'
+ actual_result=1
+ expect_result=1
+ mode=0
+ error_log='There is an error message for the log of ovsdb-server.service'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 1x
+ return 0
+ service ovsdb-server.service start
oe_test_service_ovsdb-server.sh: line 42: service: command not found
+ sed -i 's\ExecStart=/usr/share/openvswitch/scripts/ovs-ctl\ExecStart=/usr/share/openvswitch/scripts/ovs-ctl --no-mlockall\g' /usr/lib/systemd/system/ovsdb-server.service
+ systemctl daemon-reload
+ service ovsdb-server.service reload
oe_test_service_ovsdb-server.sh: line 45: service: command not found
+ CHECK_RESULT 127 0 0 'ovsdb-server.service reload failed'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='ovsdb-server.service reload failed'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'ovsdb-server.service reload failed'
+ LOG_ERROR 'ovsdb-server.service reload failed'
+ message='ovsdb-server.service reload failed'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'ovsdb-server.service reload failed'
Fri Feb  3 00:41:25 2023 - ERROR - ovsdb-server.service reload failed
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_service_ovsdb-server.sh line 46'
+ message='oe_test_service_ovsdb-server.sh line 46'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_service_ovsdb-server.sh line 46'
Fri Feb  3 00:41:26 2023 - ERROR - oe_test_service_ovsdb-server.sh line 46
+ return 0
+ service ovsdb-server.service status
+ grep 'Active: active'
oe_test_service_ovsdb-server.sh: line 47: service: command not found
+ CHECK_RESULT 1 0 0 'ovsdb-server.service reload causes the service status to change'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='ovsdb-server.service reload causes the service status to change'
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'ovsdb-server.service reload causes the service status to change'
+ LOG_ERROR 'ovsdb-server.service reload causes the service status to change'
+ message='ovsdb-server.service reload causes the service status to change'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'ovsdb-server.service reload causes the service status to change'
Fri Feb  3 00:41:26 2023 - ERROR - ovsdb-server.service reload causes the service status to change
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_service_ovsdb-server.sh line 48'
+ message='oe_test_service_ovsdb-server.sh line 48'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_service_ovsdb-server.sh line 48'
Fri Feb  3 00:41:27 2023 - ERROR - oe_test_service_ovsdb-server.sh line 48
+ return 0
+ LOG_INFO 'Finish test!'
+ message='Finish test!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish test!'
Fri Feb  3 00:41:27 2023 - INFO  - Finish test!
+ CASE_RESULT 0
+ case_re=0
+ test -z 6
+ test 6 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Fri Feb  3 00:41:28 2023 - ERROR - failed to execute the case.
+ exit 6
+ post_test
+ LOG_INFO 'start environment cleanup.'
+ message='start environment cleanup.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'start environment cleanup.'
Fri Feb  3 00:41:29 2023 - INFO  - start environment cleanup.
+ sed -i 's\ExecStart=/usr/share/openvswitch/scripts/ovs-ctl --no-mlockall\ExecStart=/usr/share/openvswitch/scripts/ovs-ctl\g' /usr/lib/systemd/system/ovsdb-server.service
+ systemctl daemon-reload
+ service ovsdb-server.service reload
oe_test_service_ovsdb-server.sh: line 56: service: command not found
+ service ovsdb-server.service stop
oe_test_service_ovsdb-server.sh: line 57: service: command not found
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z /tmp/tmp_xtj8qhz ]]
+ '[' 0 -ne 0 ']'
+ '[' 1 == 0 ']'
+ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py remove --node 1 --pkgs '' --tempfile /tmp/tmp_xtj8qhz
+ '[' 0 -ne 0 ']'
+ LOG_INFO 'Finish environment cleanup!'
+ message='Finish environment cleanup!'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Finish environment cleanup!'
Fri Feb  3 00:42:26 2023 - INFO  - Finish environment cleanup!
